src/components/container/ProviderSearch/ProviderSearch.tsx

```diff
@@ -223,7 +225,9 @@ export default function ProviderSearch(props: ProviderSearchProps) {
                 </div>
             </div>
             <div className="search-results">
-                {searchResults && searchResults.map((provider) =>
+                {searchResults && searchResults
+                    .filter(provider => provider.message !== UNAVAILABLE_PROVIDER_MESSAGE )
```

Comment:
I think maybe this filtering should be moved up to the `updateSearchResults` function where we are already doing some filtering based on provider category.  https://github.com/CareEvolution/MyDataHelpsUI/pull/475/files#diff-2019a9fcbe98ba46c5138d37c4190743b972c8d87818bc143b16b77b042a84d7R122

That way the `searchResults` will never contain these unavailable providers, so it won't cause odd behavior if we want to use the length of the results array to conditionally do something.